bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```
@@ -152,6 +152,7 @@ metadata:
         }
       ]
     capabilities: Seamless Upgrades
+    olm.skipRange: '>=1.0.0 <2.0.0'
```

@VaishnaviHire  here operator version is 0.0.4.. why do we want it to upgrade from 1.0.0 to 2.0.0... should it be something like
```suggestion
    olm.skipRange: '>=0.0.1 <0.0.4'
```

---

The manifests in this PR  are for ODH releases in operatorhub. The `z-stream` releases that we were putting out are only for Dev Builds

---

The ODH versioning for operators would be
1.x --> Old Operator
2.x --> New Operator

---

lgtm, thanks